src/telliot_feeds/sources/blockhash_aggregator.py

```diff
@@ -9,18 +9,19 @@
 import requests
 from requests import JSONDecodeError
 from requests.adapters import HTTPAdapter
+from telliot_core.apps.telliot_config import TelliotConfig
 from urllib3.util import Retry
 from web3 import Web3
 
 from telliot_feeds.datasource import DataSource
 from telliot_feeds.dtypes.datapoint import OptionalDataPoint
-from telliot_feeds.utils.cfg import mainnet_config
 from telliot_feeds.utils.log import get_logger
 
 logger = get_logger(__name__)
 
 
-cfg = mainnet_config()
+cfg = TelliotConfig()
+cfg.main.chain_id = 1
```

Comment:
Can we add a logging message that chain id 1 is being set bc required when using the rng reporter?

Reply:
sure!